package/yast2-registration.spec

```diff
@@ -58,6 +58,13 @@ BuildRequires:  yast2-update >= 3.1.19
 
 BuildArch:      noarch
 
+# FIXME: it seems can we cannot move it to macros.yast, the yast-rake-ci is not
+# installed into the chroot, the build fails...
```

Comment:
I see. For a while I thought that it may be fixable, but then I looked into [the source](https://github.com/openSUSE/obs-build/blob/5f04f59b27bc1fbd1dfa908c5287657d306fe19e/Build/Rpm.pm#L191): a 400 line function in Perl. With a `goto`.